recipe/laravel.php

```diff
@@ -105,7 +115,9 @@
 
 desc('Execute artisan storage:link');
 task('artisan:storage:link', function () {
-    run('{{bin/php}} {{release_path}}/artisan storage:link');
+    if (version_compare(get('laravel_version'), 5.2, '>')) {
```

Comment:
And as you say in the docs, you want to check if the version is `>= 5.3` or NOT `< 5.3`